resources/definitions/teraform_polaris.def.json

```diff
@@ -0,0 +1,150 @@
+        "machine_firmware_retract": 
+        { 
+            "default_value": true,
+            "value": true
+        },
+        "machine_nozzle_size": { "default_value": 12 },
+        "machine_use_extruder_offset_to_offset_coords": { "default_value": false },
+        "layer_height":
+        {
+            "minimum_value": "1",
+            "minimum_value_warning":"2",
+            "maximum_value_warning":"10",
+            "settable_per_extruder": false
+        },
+        "layer_height_0":
+        {       
+            "minimum_value": "1",
+            "minimum_value_warning":"2",
+            "maximum_value_warning":"10",
+            "settable_per_extruder": false
+        },     
+        "line_width":
+        {
+            "minimum_value": "5",
+            "minimum_value_warning": "10",
+            "maximum_value_warning": "50",
+            "settable_per_mesh": false
+        },
+        "wall_thickness":
+        {
+            "default_value": 15,
+            "value": "wall_line_width_0 if magic_spiralize else line_width",
+            "minimum_value": "line_width",
+            "minimum_value_warning": "line_width",
+            "maximum_value_warning": "4 * line_width"
+        },
+        "wall_0_wipe_dist":
+        {
+            "default_value": 0,
+            "value": 0,
+            "settable_per_mesh": false
+        },
+        "wall_min_flow": 
+        {
+            "default_value": 50,
+            "settable_per_mesh": false
+        },
+        "wall_min_flow_retract":
+        {
+            "default_value": true,
+            "settable_per_mesh": false
+        },
+        "fill_perimeter_gaps":
+        {
+            "default_value": "nowhere",
+            "settable_per_mesh": false
+        },
+        "fill_outline_gaps":
+        {
+            "default_value": false,
+            "settable_per_mesh": false
+        },
+        "ironing_enabled": 
+        {
+            "default_value": false,
+            "settable_per_mesh": false
+        },
+        "infill_sparse_density":
+        {
+            "default_value": 50
+        },
+        "infill_pattern":
+        {
+            "options":
+            {
+                "lines": "Lines",
+                "gyroid": "Gyroid"
+            },
+            "default_value": "lines"
+        },
+        "zig_zaggify_infill":
+        {
+            "value": true,
+            "settable_per_mesh": true
+        },
+        "infill_wipe_dist":
+        {
+            "enabled": false,
+            "settable_per_mesh": false
+        },
+        "infill_support_enabled":
+        {
+            "default_value": true
+        },
+        "infill_support_angle":
+        {
+            "default_value": 60
+        },
+        "retract_at_layer_change":
+        {
+            "default_value": true
+        },
+        "retraction_combing": 
+        {
+            "default_value": "off",
+            "resolve": "off"
```

Comment:
You only have a single extruder, so there is no need to change the resolve function.

Reply:
Thanks for reviewing @nallath. Working on the fix.

Reply:
Hi @nallath, mesh file sizes have been reduced to 1 meg in the latest commit